### PR TITLE
fix: #1191 Confirmation popup has too many white space

### PR DESCRIPTION
**Pull request by J0taFerreira**
- added question sign

other styling is on common repo.

### CODE REVIEW

**Comment by MCatherine1994** [comment]
Hi Jota, it looks good, just wondering if this delete confirmation popup is a little big? 
image

and shall we remove this `<br />`? I forgot to remove it in the past, thanks!!
Screen Shot 2024-03-25 at 4 49 21 PM

**Comment by MCatherine1994** [comment]
I still feel the popup width is too big... let's add the change in FAM, set the .p-dialog width to around 650px (need convert to rem), I feel that looks better
Screen Shot 2024-03-26 at 10 09 40 AM
**VS**
Screen Shot 2024-03-26 at 10 09 30 AM

How you feel about this?

**Comment by MCatherine1994** [comment]
fyi, I published the theme package v1.8.6, let's install the latest version, thanks!!

**Comment by J0taFerreira** [comment]
> I still feel the popup width is too big... let's add the change in FAM, set the .p-dialog width to around 650px (need convert to rem), I feel that looks better Screen Shot 2024-03-26 at 10 09 40 AM **VS** Screen Shot 2024-03-26 at 10 09 30 AM
> 
> How you feel about this?

did you test with the changes in max-width from the common repo? 650px makes it slightly bigger

**Comment by MCatherine1994** [comment]
> > I still feel the popup width is too big... let's add the change in FAM, set the .p-dialog width to around 650px (need convert to rem), I feel that looks better Screen Shot 2024-03-26 at 10 09 40 AM **VS** Screen Shot 2024-03-26 at 10 09 30 AM
> > How you feel about this?
> 
> did you test with the changes in max-width from the common repo? 650px makes it bigger in smaller resolutions

no I didn't try it on a smaller screen, shall we just make it a little bit smaller?

**Review by MCatherine1994 — APPROVED** [review]

File: frontend/src/assets/styles/styles.scss [hunk 1]
```diff
@@ -68,4 +68,7 @@ body {
             white-space: nowrap;
         }
     }
+    .p-dialog {
+        max-width: 40rem !important;
+    }
```
[the remark this answers is not in the document]
Comment (MCatherine1994) [reply frontend/src/assets/styles/styles.scss:73]: thanks so much!! looks good!

**Pull request merged**